presto-postgresql/src/main/java/com/facebook/presto/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -62,5 +64,18 @@ public PreparedStatement getPreparedStatement(Connection connection, String sql)
         PreparedStatement statement = connection.prepareStatement(sql);
         statement.setFetchSize(1000);
         return statement;
-    }
+   }
+
+   @Override
```

Comment:
The indentation is off here. It should be indented 4 spaces. This is the code style: https://github.com/airlift/codestyle